sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/SqlParameter.java

```diff
@@ -70,4 +71,11 @@ public SqlParameter setValue(Object value) {
         this.jsonSerializable.set("value", value);
         return this;
     }
+
+    void populatePropertyBag()
```

Comment:
code style: open braces on the same line as method definition
```suggestion
 void populatePropertyBag() {
```